### DISCUSSION TITLE
My Code Formatting Guidelines

### DISCUSSION THREAD

**Discussion by giscus** [DeclanChidlow/vale.rocks#18] [Comments Archive]
# My Code Formatting Guidelines

Code formatting, how I go about it, and why I choose to do it the way I do. This article covers indentation, string formatting, the classic tabs vs spaces debate, and much more.

https://vale.rocks/posts/my-code-formatting-guidelines

**Comment by dylan-worth** [comment]
Keep up the articles! I stumbled across your blogs when I was checking out the AutoMod bot.

**Reply by DeclanChidlow** [reply]
Thanks Dylan. Really appreciate it. :)
